app/controllers/flow/generic_handler.rb

```diff
@@ -0,0 +1,22 @@
+module Flow
+  class GenericHandler
+    def initialize(form_class)
+      @form_class = form_class
+    end
+
+    def model_from_session(session_data)
+      @form_class.new(session_data.slice(*@form_class::ATTRIBUTES.map(&:to_s))).tap { modify(_1, session_data) }
+    end
```

Comment:
In EL-419, I used a #from_session method to allow classes to override this load. Might be easier to follow than this version, WDYT? Saves a blank 'modify' part-constructor - kind of a factory pattern?

Reply:
I have taken your suggestion and added `from_session` methods to Form objects via some new concerns.